src/main/java/Duke.java

```diff
@@ -1,10 +1,409 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * A decorative prefix added to the beginning of lines.
+     */
+    public static final String LINE_PREFIX = "\t ";
+
+    public static final String HORIZONTAL_LINE = LINE_PREFIX
```

Comment:
Constants used in static, declared in capital letters. Coding standards followed